misc/build.func

```diff
@@ -8,6 +8,7 @@ variables() {
   NSAPP=$(echo ${APP,,} | tr -d ' ') # This function sets the NSAPP variable by converting the value of the APP variable to lowercase and removing any spaces.
   var_install="${NSAPP}-install"     # sets the var_install variable by appending "-install" to the value of NSAPP.
   INTEGER='^[0-9]+([.][0-9]+)?$'     # it defines the INTEGER regular expression pattern.
+  PVEHOST_NAME=$(hostname | tr a-z A-Z) # gets the Proxmox Hostname and sets it to Uppercase
```

Comment:
What's the reason to set it to uppercase?

Reply:
I would say just personal Preference. Its just a habbit of me to write "important" parts of a user message in Uppercase, just to better get the attention of the user. 
We can also do it in lowercase if this is the prefered Option.

Reply:
Switch to Default, without uppercase and lowercase. Only the Output

Reply:
Is changed to lowercase.